src/components/Modeler.vue

```diff
@@ -243,6 +257,12 @@ export default {
     minusIcon() {
       return faMinus;
     },
+    expandIcon() {
+      return faExpand;
+    },
+    compressIcon() {
+      return faCompress;
+    },
```

Comment:
These don't need to be computed properties; they will never change. They can go into the `data` object.